app/views/partials/theme/simple/layout/header.liquid

```diff
@@ -1,58 +1,66 @@
+{% liquid
+    if context.current_user.id
+        function current_profile = 'lib/current_profile', user_id: context.current_user.id	
```

Comment:
@z-x We don't want to fetch data from graphql in `theme/` files.
Layout is harder to achieve that cause we can't simply pass data from page.
We fetch profile from session in `main.liquid' and it is passed to header.

Reply:
@z-x Please add `avatar` to session in `app/views/partials/lib/commands/sessions/store_profile_in_session.liquid` then you can fetch it from variable `profile` in this file.

Something like this:
```
    "avatar":  {{ object.avatar.photo.versions.tiny | json }}
```